changelog/Microsoft.DirectoryServices.json

```
@@ -3,6 +3,40 @@
     {
       "ChangeList": [
         {
+          "Id": "b93829d7-509b-43bf-9ff7-b9a17032bae6",
+          "ApiChange": "Resource",
+          "ChangedApiName": "automaticUserConsentSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the **automaticUserConsentSettings** property to the [crossTenantAccessPolicyConfigurationDefault](https://docs.microsoft.com/graph/api/resources/crosstenantaccesspolicyconfigurationdefault?view=graph-rest-beta) resource.",
+          "Target": "crossTenantAccessPolicyConfigurationDefault"
+        },
+        {
+          "Id": "b93829d7-509b-43bf-9ff7-b9a17032bae6",
+          "ApiChange": "Resource",
+          "ChangedApiName": "automaticUserConsentSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the **automaticUserConsentSettings** property to the [crossTenantAccessPolicyConfigurationDefault](https://docs.microsoft.com/graph/api/resources/crosstenantaccesspolicyconfigurationpartner?view=graph-rest-beta) resource.",
+          "Target": "crossTenantAccessPolicyConfigurationPartner"
+        },
+        {
+          "Id": "b93829d7-509b-43bf-9ff7-b9a17032bae6",
+          "ApiChange": "Resource",
+          "ChangedApiName": "inboundOutboundPolicyConfiguration",
+          "ChangeType": "Addition",
+          "Description": "Added the [inboundOutboundPolicyConfiguration](https://docs.microsoft.com/graph/resources/inboundoutboundpolicyconfiguration?view=graph-rest-beta) resource.",
```

This should fix your changelog error
```suggestion
          "Description": "Added the [inboundOutboundPolicyConfiguration](https://docs.microsoft.com/graph/api/resources/inboundoutboundpolicyconfiguration?view=graph-rest-beta) resource.",
```